packages/jellyfish-api-core/__tests__/category/loan/loan_container.ts

```diff
@@ -2,7 +2,7 @@ import { MasterNodeRegTestContainer, StartOptions } from '@defichain/testcontain
 
 export class LoanMasterNodeRegTestContainer extends MasterNodeRegTestContainer {
   constructor () {
-    super(undefined, 'defi/defichain:HEAD-6fa1c06')
+    super(undefined, 'defi/defichain:HEAD-faf9e78')
```

Comment:
I have just updated the docker image to the latest version. Unfortunately, It breaks my previous tests.